### DISCUSSION TITLE
Thunderbird: This page is protected by Browser

### DISCUSSION THREAD

**Discussion by lamyergeier** [darkreader/darkreader#12907] [Q&A]
![image](https://github.com/darkreader/darkreader/assets/42092626/0b06582d-1217-4c8a-89a5-79f3220431ba)

I am unable to activate Dark mode or change any settings in the extension as it says (please see image above) 

> This page is protected by Browser

Is it not possible to use the extension without paying for it?

**Comment by ranibrunhilde** [comment]
I'm having this problem too on my Android Firefox. It's especially irritating that it's on ALL the Mozilla web pages.. so I get blinded when I try to download add-ons, VERY annoying.

**Reply by Myshor** [reply]
https://github.com/darkreader/darkreader#enable-dark-reader-on-restricted-pages-on-mozilla-firefox

Not sure if this can be done on mobile firefox but it's the only way to enable Dark Reader on Restricted pages in Firefox (at least not mobile).

**Comment by Myshor** [comment]
What browser and/or version of Dark Reader do you use? I see Dark Reader popup full white first time in my life. 😕
